src/app/modules/shared/components/dropdown/dropdown.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Output, Component, EventEmitter, Input } from '@angular/core';
+
+
+@Component({
+  selector: 'app-dropdown',
+  templateUrl: './dropdown.component.html',
+  styleUrls: ['./dropdown.component.scss']
+})
+export class DropdownComponent {
+
+  @Input() info: any;
+  @Output() updateInfo: EventEmitter<any> = new EventEmitter();
+
+  dataChange: any = {
+    id: '',
+    name: '',
+    description: '',
+    tenant_id: '',
+    status: '',
+    key: '',
+    _status: false,
+    btnColor: '',
+    btnIcon: '',
+    btnIconTwo: '',
+    btnName: '',
+    iconColor: ''
+  };
+
+  changePropertiesItem(item: any){
+    if (item.btnName === 'Active'){
+      this.dataChange.btnName = 'Inactive';
+      this.dataChange.btnIcon = item.btnIcon;
```

Comment:
I notice that this line:
```ts
this.dataChange.btnIcon = item.btnIcon;
```
is used in both -> if and else conditions.

As the condition is going to be the same, we can take it out even before/after the condition

Reply:
done